extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -857,11 +852,13 @@ export class AmpLightboxGallery extends AMP.BaseElement {
     if (target == this.sourceElement_) {
       return true;
     }
-    if (target.isInViewport()) {
+    // TODO: HOW TO REMOVE
+    if (false /* target.isInViewport() */) {
```

Comment:
Still TBD, right?

Reply:
Yeah, I don't have a good idea for how to do this. This PR is still in draft and I plan on splitting out some smaller changes from it (most of the things in the description's TODO, which includes how to handle this). Need to do research on lightbox gallery

Reply:
Realizing now I can defer this. Resolving